docs/server/generate_schema_documentation.py

```diff
@@ -77,10 +90,60 @@
 --------
 
 %(metadata)s
-"""
+""" + TPL_STYLES
```

Comment:
I think this might be more obvious if it were `TPL_TRANSACTION = TPL_STYLES + """...` rather than the other way around.

I was really confused what `""" + TPL_STYLE` was doing for a while 😌 .